docs/en/setup/backend/backend-alarm.md

```diff
@@ -113,6 +114,24 @@ Example as following
 }]
 ```
 
+## gRPCHook
+The alarm message will send through remote gRPC method by `Protobuf` content type. 
+The message format with following key information which are defined in `oap-server/server-alarm-plugin/src/main/proto/alarm-hook.proto`.
+
+Example as following
+```text
```

Comment:
This code language  could be `protobuf`. It will result as following:

```protobuf
message AlarmMessage {
    int64 scopeId = 1;
    string scope = 2;
    string name = 3;
    int64 id0 = 4;
    int64 id1 = 5;
    string ruleName = 6;
    string alarmMessage = 7;
    int64 startTime = 8;
}
```

Reply:
@JaredTan95 This should be fixed.

Reply:
make sense